x-pack/plugins/observability_shared/public/components/feature_feedback_button/feature_feedback_button.tsx

```diff
@@ -25,24 +25,40 @@ const getDeploymentType = (isCloudEnv?: boolean, isServerlessEnv?: boolean): str
 
 const getSurveyFeedbackURL = (
   formUrl: string,
+  formConfig?: FormConfig,
   kibanaVersion?: string,
   deploymentType?: string,
   sanitizedPath?: string
 ) => {
   const url = new URL(formUrl);
   if (kibanaVersion) {
-    url.searchParams.append(KIBANA_VERSION_QUERY_PARAM, kibanaVersion);
+    url.searchParams.append(
+      formConfig?.kibanaVersionQueryParam ?? KIBANA_VERSION_QUERY_PARAM,
```

Comment:
qq: if `formConfig?.kibanaVersionQueryParam` is an empty string, do you want to fallback to the default value (KIBANA_VERSION_QUERY_PARAM) or add the empty string to the url?

Reply:
Good question: if we have `kibanaVersionQueryParam=''` it should fallback to default I think - if there is anything else inside the `''` should use the consumer input: I mean in general if you use the component and want to override it I don't see why you would set an empty string here when you will usually use the value you want to use in the form instead 🤔 Also if you don't want the parameter at all you can achieve that by not passing the version value - then the whole parameter won't be added. Do you think we need extra validation here?

Reply:
But I am still trying to understand why we have an empty string use case here in general, I am wondering if I am not missing something or there is a case when we want to pass empty string 🤔

Reply:
I ask that because if you use `??` to check if the value is nullish (null or undefined) and someone passes an empty string this will be used. And if you use `||` and the user passes an empty string it will use the fallback value.

Reply:
Ah, I see what you mean, yeah now that I think about it we can use `||` in this case ( I am still not sure if we will have someone passing an empty string on purpose as it's a valid option in a sense but not a param we can use in the URL I think) So I changed that you are right that we should fallback to default in case of an empty string and count it as invalid.